Sources/FoundationInternationalization/Locale/Locale_ICU.swift

```diff
@@ -399,6 +400,46 @@ internal final class _Locale: Sendable, Hashable {
         }
     }
 
+    internal var identifierCapturingPreferences: String {
```

Comment:
This might be worth a comment explaining that it's not literally all preferences, but a subset of them which can be represented in the identifier. For example we don't put temperature unit in here.